templates/types/streaming/fastapi/app/engine/constants.py

```diff
@@ -0,0 +1 @@
+STORAGE_DIR = "storage"
```

Comment:
we've only got storage_dir as a constant right now. Switching it to environment variable is not a bad idea

Reply:
yes or inlining constant if it's just used once